salt/utils/stringutils.py

```diff
@@ -292,22 +292,31 @@ def __build_parts(text):
     return r'(?m)^{0}$'.format(regex)
 
 
-def expr_match(line, expr):
+def expr_match(val, expr):
```

Comment:
What's the reasoning behind changing this variable name? This is a public facing function so I don't think we should change it unnecessarily.